part1/part1.md

```diff
@@ -109,6 +177,9 @@ false|false|false|                       |
   * Write an `until` loop that prints out all of the numbers that are less than 20 and are not divisible by 6.
```

Comment:
Yes, but we use them a lot in the solutions, and I think they make a lot of code easier to parse for newbies. I'd go over them early on, especially for people unfamiliar with Ruby.